openedx/core/lib/blockstore_api/methods.py

```diff
@@ -144,6 +144,18 @@ def delete_collection(collection_uuid):
     api_request('delete', api_url('collections', str(collection_uuid)))
 
 
+def get_bundles(uuids=None):
+    """
+    Get the details of all bundles
+    """
+    if uuids is None:
+        uuids = []
+    version_url = api_url('bundles') + '?uuid=' + ','.join(map(str, uuids))
+    response = api_request('get', version_url)
+    # build bundle from response, convert map object to list and return
+    return [*map(_bundle_from_response, response)]
```

Comment:
style nit: In edx-platform we tend to prefer list-comprehension syntax over `map`, `filter`, etc.